fv3core/tests/savepoint/translate/translate_tracer2d1l.py

```diff
@@ -1,12 +1,12 @@
 import pytest
 
-import fv3core.stencils.fvtp2d
-import fv3core.stencils.tracer_2d_1l
 import pace.dsl
 import pace.dsl.gt4py_utils as utils
+import pace.fv3core.stencils.fvtp2d
+import pace.fv3core.stencils.tracer_2d_1l
```

Comment:
can we change these to 
`from pace.fv3core.stencils.fvtp2d import FiniteVolumeTransport` and
`from pace.fv3core.stencils.tracer_2d_1l import TracerAdvection`

as in the other translate classes?

Reply:
I guess it's not really standard across the translate test files, but it would be nice to cleanup at some point

Reply:
I'd much rather we put these all in the same namespace, e.g. `from pace.fv3core.stencils import FiniteVolumeTransport, TracerAdvection`